relayer/relay.go

```diff
@@ -167,6 +168,8 @@ func (r *Relayer) StartRouter(ctx context.Context, flushInterval time.Duration)
 		case <-routeTimer.C:
 			// processMessage starting working on all the runtime Messages
 			r.processMessages(ctx)
+		case <-heightTimer.C:
+			r.SaveChainsBlockHeight(ctx)
```

Comment:
Won't the saving interval of 5 minutes pose any problem? any specific reason to set it for this high value ?

Reply:
Its arbitrary value, going back to previous height wont be a problem so that we made sure that we catch all of them just in case.